.github/workflows/ci.yml

```diff
@@ -4,6 +4,10 @@ on:
   # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
   push:
   pull_request:
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
```

Comment:
The link should be https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency since we are setting the concurrency on the workflow level(not jobs.)
```suggestion
# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency
```